tests/test_ellipKPM.py

```diff
@@ -74,7 +75,7 @@ def test_te1(self):
             result = task.run('TE1', {'i': [CalibratedCatalog(catalog), ]})
             log.debug('result: ', result)
             log.debug('expected: ', expected)
-            self.assertAlmostEqual(result.measurement, expected, places=10)
+            np.testing.assert_almost_equal(result.measurement.quantity, expected.quantity, decimal=5)
```

Comment:
What's the reason to switch to numpy testing here? I would have thought it made sense for arrays, but that doesn't seem to be the case.

Reply:
Perhaps, you could add the reason in the body of the commit message? https://developer.lsst.io/work/flow.html?highlight=commit#writing-commit-message-body-content

Reply:
Sure. FYI it's because `assertAlmostEqual` doesn't actually work for `lsst.verify.measurement.Measurement`objects (since it doesn't have subtraction defined). Unfortunately `assertAlmostEqual` doesn't work with `astropy.quantity.Quantity` values either, so I had to both extract the quantity _and_ use numpy's almost equal. 😞